source/retryable-reads/tests/changeStreams-client.watch-serverErrors.yml

```
@@ -0,0 +1,169 @@
+database_name: &database_name "retryable-reads-tests"
+collection_name: &collection_name "coll"
+
+data:
+    - {_id: 1, x: 11}
+    - {_id: 2, x: 22}
+    - {_id: 3, x: 33}
+    
+minServerVersion: '4.1.6' # 4.2
+
+tests:
+    -
+        description: "client.watch succeeds after InterruptedAtShutdown"
+        clientOptions:
+            retryReads: true
+        failPoint: &failCommand_failPoint
+            configureFailPoint: failCommand
+            mode: { times: 1 }
+            data: { failCommands: [aggregate], errorCode: 11600 }
+        operation: &retryable_operation
+            name: watch
+            object: client
+        expectations:
+            - &retryable_command_started_event
+                command_started_event:
+                    command:
+                        aggregate: 1
+                        cursor: {}
+                        pipeline: [ { $changeStream: { fullDocument: "default" } } ]
+                    database_name: *database_name
```

Db should be admin for cluster level changes.
```suggestion
                    database_name: admin
```